### ISSUE TITLE
Cannot transform string to number

### ISSUE THREAD

**Issue by CodeSnooker** [typestack/class-transformer#179]
Example:-

```Typescript
import { plainToClass } from 'class-transformer';

export class ListQueryDto {
    @IsDefined()
    readonly limit: number = 20;
}

const tempQuery: ListQueryDto = plainToClass(ListQueryDto, { limit: '100' });
console.log('Modified => ', tempQuery);
```

When we print the result, the limit is printed as string instead as a number.

**Current Output**
`Modified =>  ListQueryDto { limit: '100' }`
 
**Expected Output**
`Modified =>  ListQueryDto { limit: 100 }`

**Comment by cojack** [comment]
@CodeSnooker it won't work this way, because types are gone after code transpilation, so to achieve your goal you have to use `@Transform` like that:

```
import { plainToClass } from 'class-transformer';

export class ListQueryDto {
    @Transform(value => parseInt(value, 10), { toClassOnly: true })
    readonly limit: number = 20;
}

const tempQuery: ListQueryDto = plainToClass(ListQueryDto, { limit: '100' });
console.log('Modified => ', tempQuery);
```

**Comment by NoNameProvided** [comment]
> @CodeSnooker it won't work this way, because types are gone after code transpilation,

That's not correct, we have type information on builtins. 

I can reproduce this with the following snipet: 

```ts
import 'reflect-metadata';
import { plainToClass, Type } from 'class-transformer';

export class ListQueryDto {
  readonly limit: number;
}

const tempQuery: ListQueryDto = plainToClass(ListQueryDto, { limit: '100' });
console.log('Modified => ', tempQuery);
```

For now, a quick fix can be to simply specify the expected type explicitly via the `@Type` decorator: 

```ts
export class ListQueryDto {
  @Type(() => Number) 
  readonly limit: number;
}
``` 

But this should not be required and needs to be fixed.

**Comment by NoNameProvided** [comment]
Will be released as `0.2.2`
